lib/net.js

```diff
@@ -234,9 +238,12 @@ function Socket(options) {
 }
 util.inherits(Socket, stream.Duplex);
 
+// Refresh existing timeouts.
```

Comment:
If your run `active()` or `_unrefActive()` on an existing timer object it removes it from the underlying list and re-appends it to the appropriate timer pool, refreshing the timer. (Same duration but from "now" rather than when it was first scheduled.)

@addaleax Can you think of a wording that would be more helpful based on that?